### ISSUE TITLE
Vim indentation

### ISSUE THREAD

**Issue by junwei17** [nus-cs2030/2021-s2#27]
are there any commands where i can auto indent my code in Vim?

**Comment by jiajun-seah** [comment]
Yes, you can enter the command "gg=G" to auto indent in Vim. You should avoid this if you're coding in Python though!

**Comment by miao-tianjia** [comment]
For me, I changed my Vim settings, so it will auto-indent and show the line number when I use vim to code (you can try to google 'set vim settings' and there are complete guide and code for your reference! Just copy paste into your own vimrc file and that is done!)

**Comment by gableejh** [comment]
Hi, before you enter "gg=G", ensure that you have entered the following commands into your vimrc file. 

![107111951-f92c3200-688e-11eb-9686-273180086a0c](https://user-images.githubusercontent.com/77229491/107180545-e2fbae80-6a13-11eb-918d-4e0745c8f9d1.png)

**Comment by lozhaowei** [comment]
Yes! you can enter "gg-G" (the capitalisation matters) when in command mode. All your code in that page will be indented according to your brackets. You can even use it to check if you have any unclosed brackets if you see any weird indentations!

**Comment by rendao1010** [comment]
Hi, I believe that there is a command "gg=G" in vim that helps to solve this issue. Regardless, it is good practice to ensure that your code is indented appropriately by yourself as you write your code! Hope this helps.
